PROTOCOL.agent

```diff
@@ -49,9 +49,11 @@ Where a constraint consists of:
 
 	string		from_username (must be empty)
 	string		from_hostname
+	string		reserved
 	keyspec[]	from_hostkeys
 	string		to_username
 	string		to_hostname
+	string		reserved
 	keyspec[]	to_hostkeys
 
```

Comment:
```suggestion
string reserved
```

There is also this reserved field:
https://github.com/openssh/openssh-portable/commit/5e950d765727ee0b20fc3d2cbb0c790b21ac2425#diff-62fc3a49d4e4e6c55819579611e5a53767c9dcc66aafa74fd3aa067ea0b5d31dR495

Reply:
Thanks for the double-check @baloo! I've squashed your change with mine and added you as `Co-authored-by` :)